src/content/wiki/tutorials/getting-started.mdoc

```diff
@@ -34,7 +34,7 @@ You should also gather as many resources as you can-wood, as well as cobble, coa
 
 ### Step 1: Supply Camp/Supply Ship
   
-Once you have chosen a location for your colony, you must place a {% item name="build_tool" /%}, the remaining camp/ship is just decoration. You can leave it or tear it down.
+Once you have chosen a location for your colony, you must place a {% item name="supply_camp_and_ship" /%}. Go inside the ship/camp and aquire the {% item name="build_tool" /%} . The remaining camp/ship is just decoration. You can leave it or tear it down.
```

Comment:
- "acquire"
- Extra space after build tool
- Not all styles continue to have a build tool/town hall inside of the supply camp as they can be crafted separately. So maybe state "and acquire the build tool if the supply camp contains one, else craft it"

Reply:
> Not all styles continue to have a build tool/town hall inside of the supply camp as they can be crafted separately. So maybe state "and acquire the build tool if the supply camp contains one, else craft it"

Afaik having a town hall and build tool in the supply camp and ship is a requirement of those, so this should be true for all of them